models/automations/view-automation-history.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: View an automation's history
+---
+
+<Note>
+This feature requires an Enterprise license. It is available only for:
+- W&B Multi-tenant Cloud
+- W&B Dedicated Cloud
+- W&B Self-Managed v0.75.0+
+</Note>
+
+This page describes how to view and understand the execution history of your W&B [automations](/models/automations), including what triggered an automation, what actions were taken, and whether they succeeded or failed.
+
+## Overview
+
+Each executed automation generates a record that includes:
+- **Execution timestamp**: When the automation was triggered.
+- **Triggering event**: The specific event triggered the automation.
+- **Status**: The execution's status. See [Execution status](#execution-status).
+- **Action details**: Information about what action was performed, such as notifying a Slack channel or running a webhook.
+- **Result details**: Additional information, if any, about the final outcome of executing the automation, including the error for a failed execution.
+
+## View automation history
+
+You can view automation history from multiple locations in the W&B App:
+
+### From the Automations tab
```

Comment:
Confused about this section title, the "From the automation details page" also requires clicking on a tab to view automation history. Suggest renaming sections. Or grouping "registry" vs "project" flow since that's how the product is split it.